testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -264,6 +264,34 @@ def test_deprecated_warning(self, hole, attrname):
             value = getattr(hole, attrname)
             assert value is hole.results[attrname]
 
+
+class TestOserror(object):
```

Comment:
If you make this its own class, please follow PEP8. Python 3+ also doesn't need to directly subclass `object` for a normal class.

Elsewhere in the tests there should be a `pytest` skip condition (`skipif`) that checks if the `hole` executable is found. If not, then the HOLE tests don't run. This isn't triggering tests on Linux so far, but to be safe, could you please add that condition here so that the tests only run if HOLE is found? (It might also be the reason for the failures on Windows).